quora-api/src/main/java/com/upgrad/quora/api/controller/AdminController.java

```diff
@@ -27,7 +27,7 @@
      */
     @RequestMapping(method = RequestMethod.DELETE, path = "/admin/user/{userId}", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
     public ResponseEntity<UserDeleteResponse> deleteUser(@RequestHeader("authorization") final String authorization, @PathVariable String userId) throws AuthorizationFailedException, UserNotFoundException {
-        String deletedUserUUID = authenticationService.checkUserByAuthToken(authorization, userId);
+        String deletedUserUUID = authenticationService.deleteUserByAuthToken(authorization, userId);
```

Comment:
This code pushed and merged can you pull once

Reply:
Will merge the code. I shall try use this for checking the UUID.